models/linkages/R/write.restart.LINKAGES.R

```diff
@@ -77,10 +70,18 @@ write.restart.LINKAGES <- function(out.dir, runid, time, settings, analysis.vec,
   }
   
   spp.params <- spp.params.default[spp.params.save,]
+  
+  biomass_function<-function(dbh){ #kg/tree
+    .1193 * dbh^2.393 + ((slta+sltb*dbh)/2)^2 * 3.14 * fwt * frt * .001
+  }
+  merit<-function(dbh){
+    (b_obs - biomass_function(dbh))^2
```

Comment:
How about outside the function but inside the package and start the function name with a . (e.g. .merit) which @rykelly claims will keep it from being exported.